src/olympia/lib/es/management/commands/reindex.py

```diff
@@ -95,6 +80,17 @@ def unflag_database():
     unflag_reindexing_amo()
 
 
+# FIXME: time limits, need to pass
```

Comment:
follow-up issue or something you meant to fix in this issue?

Reply:
I forgot to fix it here... but I'm not sure it matters - I don't think we need a specific limit. I'm inclined to remove it.

Reply:
We currently have in our settings:
```python
CELERY_TIME_LIMITS = {
    # The reindex management command can take up to 3 hours to run.
    'olympia.lib.es.management.commands.reindex': {
        'soft': 10800, 'hard': 14400},
}
```

We know from last week it's not true - it may have been in the past, but now that the users, collections and compat data have been removed, it's way faster. 

More importantly, I don't think this can work now that the global indexing task has been replaced with a group - if we apply a limit to a group, it will be applied to each individual task, but not to the group itself (the doc say any options pass will be `applied to all tasks in the group`)

TL;DR: I'll remove this.

Reply:
1a3a8eebe52b88cd445f2ae798390a861783b0c3